static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -1,35 +1,79 @@
-import {useState} from 'react';
+import {useCallback, useEffect, useState} from 'react';
 import styled from '@emotion/styled';
 
+import {promptsCheck, promptsUpdate} from 'sentry/actionCreators/prompts';
 import {Button} from 'sentry/components/button';
 import Card from 'sentry/components/card';
+import {openConfirmModal} from 'sentry/components/confirm';
+import {DropdownMenu, MenuItemProps} from 'sentry/components/dropdownMenu';
 import ExternalLink from 'sentry/components/links/externalLink';
-import {Tooltip} from 'sentry/components/tooltip';
-import {IconCommit, IconGithub, IconInfo, IconMail} from 'sentry/icons';
+import QuestionTooltip from 'sentry/components/questionTooltip';
+import {IconCommit, IconEllipsis, IconGithub, IconMail} from 'sentry/icons';
 import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import {MissingMember, Organization} from 'sentry/types';
+import {promptIsDismissed} from 'sentry/utils/promptIsDismissed';
+import useApi from 'sentry/utils/useApi';
 import withOrganization from 'sentry/utils/withOrganization';
 
 type Props = {
   missingMembers: {integration: string; users: MissingMember[]};
-  onSendInvite: (email: string) => Promise<void>;
+  onSendInvite: (email: string) => void;
   organization: Organization;
 };
 
 export function InviteBanner({missingMembers, onSendInvite, organization}: Props) {
   // NOTE: this is currently used for Github only
-  // TODO(cathy): include snoozing, docs link
   const [sendingInvite, setSendingInvite] = useState<boolean>(false);
+  const [showBanner, setShowBanner] = useState<boolean>(false);
+
+  const api = useApi();
+  const integrationName = missingMembers?.integration;
+  const promptsFeature = `${integrationName}_missing_members`;
+
+  const snoozePrompt = useCallback(async () => {
+    await promptsUpdate(api, {
+      organizationId: organization.id,
+      feature: promptsFeature,
+      status: 'snoozed',
+    });
+
+    setShowBanner(false);
```

Comment:
i feel like you can optimistically update state and make the request in the background

Reply:
what does that mean? show it first and then hide it if necessary?

Reply:
that way the click to close the banner feels fast